package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "clean": "rm -rf cache artifacts typechain-types",
     "build": "forge build",
-    "lint": "solhint -w 130 -c .solhint-src.json './src/**/*.sol' && solhint -w 245 -c .solhint-test.json './test/**/*.sol'",
+    "lint": "solhint -w 140 -c .solhint-src.json './src/**/*.sol' && solhint -w 290 -c .solhint-test.json './test/**/*.sol'",
```

Comment:
Do we need to revert this? You can run yarn lint to set the value

Reply:
Not sure, just noticed that that was one of the changes in the PR that enshrined the token receiver functions, so I reverted. I can just go ahead and keep these values the same 👍